sources/packages/backend/apps/db-migrations/src/sql/EducationPrograms/Add-credential-types-aviation-column.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE
+  sims.education_programs
+ADD
+  COLUMN credential_types_aviation jsonb;
+
+COMMENT ON COLUMN sims.education_programs.credential_types_aviation IS 'A jsonb column to store various credential types related to aviation programs.';
+
+ALTER TABLE
+  sims.education_programs_history
+ADD
+  COLUMN credential_types_aviation jsonb;
+
+COMMENT ON COLUMN sims.education_programs_history.credential_types_aviation IS 'A jsonb column to store various credential types related to aviation programs.';
```

Comment:
Can we be less technical on the comment. i.e. excluding the jsonb. It can be simple as `Aviation program credential types.`